src/io.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const prompt = require('prompt-sync')();
+
+const question = () => {
+  const numbers = prompt('Enter 4 numbers: ');
+
+  if (numbers.length !== 4 || !numbers.match(/^[0-9]+$/)) {
```

Comment:
you could cover both evaluations within regex
```suggestion
  if (!numbers.match(/^\d{4}$/)) {
```